server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/AbstractRecipientRewriteTableMailet.java

```diff
@@ -44,6 +50,54 @@
  * operations based on RecipientRewriteTable implementations
  */
 public abstract class AbstractRecipientRewriteTableMailet extends GenericMailet {
+    private static final Predicate<Mapping> noneDomain() {

```

Comment:
final on function is a bit pointless, but you could create instance of prediates : 
```
private static final Predicate<Mapping> noneDomain = new Predicate ...
```